docs/_docs/plugins.md

```diff
@@ -58,6 +58,21 @@ You have 3 options for installing plugins:
 
 <div class="note info">
   <h5>
+    The <code>:jekyll_plugins</code> group in a <code>Gemfile</code>
+  </h5>
+  <p>
+    Jekyll treats this particular group of gems(s) with a special love. Any gem
```

Comment:
```
Jekyll gives this particular group of gems different treatment.
```

instead of
```
Jekyll treats this particular group of gems(s) with a special love.
```

Reply:
or how about
```
Jekyll treats this particular group of gems(s) differently
```
?

Reply:
That's fine too. Just need one `s` on `gem(s)` though. 😄